<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="73940" started-at="2019-12-26T19:00:18 IST" finished-at="2019-12-26T19:01:32 IST">
    <groups>
    </groups>
    <test name="TestLogin" duration-ms="73940" started-at="2019-12-26T19:00:18 IST" finished-at="2019-12-26T19:01:32 IST">
      <class name="com.actitime.test.TestLicence">
        <test-method status="PASS" signature="launchApp()[pri:0, instance:com.actitime.test.TestLicence@43814d18]" name="launchApp" is-config="true" duration-ms="60423" started-at="2019-12-26T19:00:18 IST" finished-at="2019-12-26T19:01:19 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method status="FAIL" signature="testLicence()[pri:0, instance:com.actitime.test.TestLicence@43814d18]" name="testLicence" duration-ms="12298" started-at="2019-12-26T19:01:19 IST" finished-at="2019-12-26T19:01:31 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [actiTIME 2020.1 Proj] but found [actiTIME 2020.1 Pro]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [actiTIME 2020.1 Proj] but found [actiTIME 2020.1 Pro]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.actitime.pom.LicensePage.verifyLicencesVersion(LicensePage.java:21)
at com.actitime.test.TestLicence.testLicence(TestLicence.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLicence -->
        <test-method status="PASS" signature="teardownApp(org.testng.ITestResult)[pri:0, instance:com.actitime.test.TestLicence@43814d18]" name="teardownApp" is-config="true" duration-ms="1187" started-at="2019-12-26T19:01:31 IST" finished-at="2019-12-26T19:01:32 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testLicence status=FAILURE method=TestLicence.testLicence()[pri:0, instance:com.actitime.test.TestLicence@43814d18] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownApp -->
      </class> <!-- com.actitime.test.TestLicence -->
    </test> <!-- TestLogin -->
  </suite> <!-- Suite -->
</testng-results>
